### PR TITLE
Release/3.0.0

### PR DESCRIPTION
**Pull request by KtorZ**

### CODE REVIEW

File: .coverage/cardano-addresses.overlay [hunk 1]
```diff
@@ -20,10 +20,6 @@ module "Cardano.Address.Derivation" {
   tick "getIndex" [Record Accessor];
 }
 
-module "Cardano.Address.Errors" {
-  tick function "showsPrec" [Show instance];
-}
-
```
Comment (KtorZ) [inline .coverage/cardano-addresses.overlay:26]: That module no longer exists.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ jobs:
 
     - name: 🔨 Build & Test
       run: |
-        stack --no-terminal test --bench --haddock --no-haddock-deps --no-run-benchmarks --flag cardano-addresses:release --flag cardano-addresses-cli:release
+        stack --no-terminal test --no-run-tests --bench --haddock --no-haddock-deps --no-run-benchmarks --flag cardano-addresses:release --flag cardano-addresses-cli:release
```
Comment (KtorZ) [inline .github/workflows/main.yml:43]: We are already running the tests right after as part of the coverage calculation yet, we still want to build the doc and with the proper "release" flag, but no need to run the tests twice.

Comment (piotr-iohk) [reply .github/workflows/main.yml:43]: ok, I was just about to ask...

**Review by piotr-iohk — APPROVED** [review]
lgtm

**Pull request merged**